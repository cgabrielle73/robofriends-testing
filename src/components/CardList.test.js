import { shallow } from "enzyme";
import React from 'react';
import CardList from "./CardList";

test(' card component', () => {
    const mock = [
        {
            id: 1,
            name: 'robo',
            email: 'robo@gmsd.com'
        },
        {
            id: 2,
            name: 'robow',
            email: 'roboww@gmsd.com'
        },
    ]
    expect(shallow(<CardList robots={mock}/>)).toMatchSnapshot();
})