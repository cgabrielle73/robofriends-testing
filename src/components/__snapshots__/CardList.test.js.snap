// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` card component 1`] = `
<div>
  <Card
    email="robo@gmsd.com"
    id={1}
    key="0"
    name="robo"
  />
  <Card
    email="roboww@gmsd.com"
    id={2}
    key="1"
    name="robow"
  />
</div>
`;
